CHANGELOG.md

```diff
@@ -6,11 +6,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
 
-This version of the dashboard requires Cumulus API >= v19.2.0-alpha.1 (TBD API release)
```

Comment:
Keep this line "This version of the dashboard..." under `## [Unreleased]`, it will be updated when we have a core release.
Other than this, my comments are addressed. Thanks.

Reply:
I added it back in. Thanks @jennyhliu.